### PR TITLE
selenium-4 changes

### PR DESCRIPTION
**Pull request by RutvikChandla**

### CODE REVIEW

**Comment by asambstack** [comment]
Same review comments as https://github.com/browserstack/php-selenium-browserstack/pull/2/files.
Caps for sel4 looks good.

**Comment by RutvikChandla** [comment]
@asambstack Please review.

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
+# php-selenium-browserstack
 
+# php-selenium-browserstack
```
Comment (asambstack) [inline README.md:3]: Why is there `php-selenium-browsertsack` twice?

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+	"browserVersion" => "latest",
+);
+
+// Set you credentails
+$BROWSERSTACK_USERNAME = "BROWSERSTACK_USERNAME";
+$BROWSERSTACK_ACCESS_KEY = "BROWSERSTACK_ACCESS_KEY";
+```
+
+## To run tests
+---
```
Comment (asambstack) [inline README.md:49]: This line can be removed.

**Review by asambstack — COMMENTED** [review]
Minor comments in readme. Rest LGTM.

**Review by asambstack — APPROVED** [review]
LGTM

**Pull request merged**